Assets/Localization/StringTables/Internal_RSC_en.asset

```diff
@@ -11195,12 +11195,12 @@ MonoBehaviour:
       reptilian nature, Argonians do not tire[/center]
 
       easily when swimming.
-      While many Argonians have[/center]
+      Many Argonians have[/center]
```

Comment:
This removes the opposition in the original sentence between thieves/spellcasters on one hand and warriors in the other, what justifies that change?

Reply:
This change goes hand in hand with the following change. The original line is "While many Argonians have successfully mastered the arts of thievery and spellcasting, there are some regarded well as warriors."
The 'while many' phrasing is often used to indicate "on the one hand X, but this is undermined by Y." A better phrasing would indicate "sometimes X, but also sometimes Y" as it's not ambiguously negative, so the tone can better match the rest of the statement in my opinion. Likely one of the many cases where the original wording was changed by modifying a couple words (e.g. 'they are terrible warriors'->'there are some regarded well as warriors'), without altering other parts of the sentence to better work with the change.

Reply:
I think there's yet another way to understand the opposition in the original sentence, the author trying to insist on the stereotype of argonian as thieves: "usually X, but on occasion Y". Or, exception that proves the rule: argonian warriors, having rules of conduct incompatible with theft, not matching folks expectations concerning argonians.
That's how I understood the original line, and I feel it's lost in the rewrite. Maybe other reviewers will read it differently.

Reply:
My understanding of TES is that Khajiit (cat burglars) are the stereotypical thieves rather than Argonians, and this sentence says 'thievery and spellcasting'. It seems to me that the first half of the sentence has a negative tone, which conflicts with the neutral second half. I rewrote the whole thing to be neutral, although it could've also been rewritten to be thoroughly negative.

I'll defer to a 3rd person to decide if this sounds better: "While many Argonians have successfully mastered the arts of thievery and spellcasting, only some are regarded well as warriors."
What's currently in the PR says "Many Argonians have successfully mastered the arts of thievery and spellcasting, and some are regarded well as warriors."

Reply:
Ah yes, I confused the two; Well, personally I'm fine with that last rewrite

Reply:
DFU 1.0.0 text:
> Argonians hail from the province of Black Marsh. You are part of a highly-evolved race of reptiles, known for their intelligence, agility, and speed. Because of their reptilian nature, Argonians do not tire easily when swimming. While many Argonians have successfully mastered the arts of thievery and spellcasting, there are some regarded well as warriors. Is your character to be an Argonian?

Current rewrite:
> Argonians hail from the province of Black Marsh. You are part of a highly-evolved race of reptiles, known for their intelligence, agility, and speed. Because of their reptilian nature, Argonians do not tire easily when swimming. Many Argonians have successfully mastered the arts of thievery and spellcasting, and some are regarded well as warriors. Is your character to be an Argonian?

My proposed rewrite:
> Argonians hail from the province of Black Marsh. You are part of a highly-evolved reptilian people, extremely intelligent, agile and quick. Much like their reptile kin, Argonians do not tire easily when swimming. While some are regarded well as warriors, Argonians are better-known for successfully mastering the arts of thievery and the arcane alike. Is your character to be an Argonian?

This rewrite follows a formula much closer to that of the other races:
> Race hail from the province of Someplace. You are part of a(n) adjective-skinned people, (known to be) extremely/highly <pick 2 or 3 adjectives from: strong, hardy, willful, intelligent, agile, quick>. Racial characteristic. Class recommendations. Is your character to be a(n) race?

Reply:
The racial descriptions subtly differ and don't seem to follow the template that strictly. I'd think it'd be preferable to leave the writing's 'flavor' alone as much as possible in the main DFU release, only changing as little as is required to correct any errors present (or improve punctuation). Perhaps that was your point, but I judged that the minor impact on style was justified in order to resolve the contradiction in tone.

Reply:
> Perhaps that was your point

Pretty much. Personally I believe we should stick to (relatively) objective grammar, spelling and punctuation corrections. Tone and other stylistic corrections should be left aside.

From a pragmatic POV, I don't think the tone of the sentence is something to be corrected in the first place. The paragraph it belongs to switches between talking to the player's character:
"**You are** part of a highly-evolved race of reptiles", and talking to the player directly: "Is **your character** to be an Argonian?"

It is with this insight that I think that the sentence doesn't have a _real_ contradiction. The intended tone of the sentence is neutral throughout.

> While many Argonians have successfully mastered the arts of thievery and spellcasting, there are some regarded well as warriors.

Is supposed to be read as "while the Argonian race is best for thief-like and spellcasting characters, it is also a decent choice for warriors". We have to remember that Daggerfall characters were originally supposed to receive [attribute buffs and debuffs](https://web.archive.org/web/20190813050705/https://en.uesp.net/wiki/Daggerfall:Races) based on race and gender, on top of those of their class.

Reply:
I found some sources online that used the "while many... there are some" phrasing, and I've grown ambivalent about that wording, so I reverted the change. This wasn't really meant to be a broader editing pass, and as pointed out, there are additional awkward wordings I didn't touch.